src/core/server/rendering/views/template.tsx

```diff
@@ -149,7 +149,7 @@ export const Template: FunctionComponent<Props> = ({
    * @returns a loading bar component or no loading bar component
    */
   const renderBrandingEnabledOrDisabledLoadingBar = () => {
-    if (customLoadingLogo() && !injectedMetadata.branding.loadingLogo.defaultUrl) {
+    if (customLoadingLogo() && !injectedMetadata.branding.loadingLogo?.defaultUrl) {
```

Comment:
can we reuse the one you defined on line 99?

Reply:
Nice catch thx~